src/lib/seam/thermostats/climate-setting-schedules/use-delete-climate-setting-schedule.ts

```diff
@@ -3,16 +3,19 @@ import {
   type UseMutationResult,
   useQueryClient,
 } from '@tanstack/react-query'
-import type { ClimateSettingScheduleDeleteRequest, SeamError } from 'seamapi'
+import type {
+  ClimateSettingScheduleDeleteRequest,
+  ClimateSettingScheduleDeleteResponse,
+  SeamError,
+} from 'seamapi'
 
 import { NullSeamClientError, useSeamClient } from 'lib/seam/use-seam-client.js'
 
 export type UseDeleteClimateSettingScheduleMutationParams =
   ClimateSettingScheduleDeleteRequest
 
-export interface UseDeleteClimateSettingScheduleData {
-  ok: true
-}
+export type UseDeleteClimateSettingScheduleData =
+  ClimateSettingScheduleDeleteResponse
```

Comment:
It looks like in the other hooks we explicitly define the type here and do not use `*Response`. I believe the reasoning for this is better editor / type dx. I think we should keep this pattern consistent in the code base, but it's something we might be able to improve on with `@seamapi/types`

```suggestion
export type UseDeleteClimateSettingScheduleData = void
```

Reply:
Sorry, I'm getting
```
/home/runner/work/react/react/src/lib/seam/thermostats/climate-setting-schedules/use-delete-climate-setting-schedule.ts
Error:   13:51  error  void is only valid as a return type or generic type argument  @typescript-eslint/no-invalid-void-type

```

Can we keep it as it was before?

Reply:
I get the same thing if I pass in `void` to `UseMutationResult<>`

Reply:
Sure. This is annoying but I get it.